cli/flags.rs

```diff
@@ -35,6 +34,11 @@ pub enum DenoSubcommand {
   Completions {
     buf: Box<[u8]>,
   },
+  Cover {
+    files: Vec<PathBuf>,
+    ignore: Vec<PathBuf>,
```

Comment:
Maybe --ignore to ignore files (which are coverage data files) and --exclude to exclude source files?